tests/datasources/test_chirps.py

```diff
@@ -0,0 +1,487 @@
+"""Tests for the Chirps module."""
+import calendar
+from datetime import date
+
+import cftime
+import numpy as np
+import pytest
+import xarray as xr
+from xarray.coding.cftimeindex import CFTimeIndex
+
+from aatoolbox import ChirpsDaily, ChirpsMonthly, GeoBoundingBox
+
+DATASOURCE_BASE_DIR = "chirps"
```

Comment:
Either make all the globals private or public. For tests I am usually lazy and make everything public since it doesn't really matter.